mitodl/micromasters
# Protected final grade audit

#### What are the relevant tickets?
fixes https://github.com/mitodl/micromasters/issues/2000

#### What's this PR do?
- It adds a rule `DO INSTEAD NOTHING` on `grades_finalgradeaudit` for update and delete. 

#### How should this be manually tested?
- run `docker-compose run web ./manage.py migrate  grades`
- docker-compose run web ./manage.py shell
```
from grades.models import FinalGradeAudit
- In [4]: FinalGradeAudit.objects.all().delete()
- Out[4]: (0, {'grades.FinalGradeAudit': 0})
```
- check everything is there on http://192.168.99.100:8079/admin/grades/finalgradeaudit/

@pdpinch
